src/hunspell/affentry.cxx

```diff
@@ -217,7 +217,7 @@ struct hentry* PfxEntry::checkword(const char* word,
   // and if there are enough chars in root word and added back strip chars
   // to meet the number of characters conditions, then test it
 
-  int tmpl = len - appnd.size(); // length of tmpword
+  int tmpl = len - int(appnd.size()); // length of tmpword
```

Comment:
This is C++ style, but much shorter. The style is like an explicit constructor call. I don't find any advantage in static_cast compared to this. That's just longer and harder to read.